Update docker image automatically (#788)
demisto/demisto-sdk: CHANGELOG.md

* Added support for *prev-ver* flag in **lint** and **secrets** commands.
* Added support for *text* flag to **update-release-notes** command to add the same text to all release notes.
* Fixed an issue where **validate** did not recognize added files if they were modified locally.

> * Add enhancement to **update-release-notes**: update release notes automatically when upgrading docker image.

Reviewer: ```suggestion
* The **update-release-notes** command will now automatically add docker image update to the release notes.
```
